templates/web/pages/org_profile/org_profile.tt.html

```diff
@@ -14,6 +14,37 @@
 	<p>&rarr; <a href="[% link %]">[% lang("official_site") %]</a></p>
 [% END %]
 
+[% IF org_members.size %]
+<h2>[% lang("organization_members") %]</h2>
+	<p>[% lang("number_of_members") %]: [% org_members.size %]</p>
```

Comment:
```suggestion
	<p>[% lang("number_of_members") %][% sep %]: [% org_members.size %]</p>
```

This will add a space in some languages (like French) where we need a space before the : character.